packages/php-wasm/node/src/test/php-networking.spec.ts

```diff
@@ -21,7 +21,7 @@ describe.each(SupportedPHPVersions)(
 			expect(text).toEqual('response from express');
 		});
 	},
-	1000
+	5000
```

Comment:
Why change this value to 5000?

Reply:
@adamziel This change was originally related to the e2e crash, which I suspected to be resource related, but I quickly determined that this wasn't the cause once I was able to get things locally, so I've reverted this change.